metrics-tracing-context/src/tracing_integration.rs

```diff
@@ -0,0 +1,133 @@
+//! The code that integrates with the `tracing` crate.
+
+use metrics::Label;
+use std::{any::TypeId, marker::PhantomData};
+use tracing_core::span::{Attributes, Id, Record};
+use tracing_core::{field::Visit, Dispatch, Field, Subscriber};
+use tracing_subscriber::{layer::Context, registry::LookupSpan, Layer};
+
+struct Labels(Vec<Label>);
+
+impl Visit for Labels {
```

Comment:
Note to myself here: we should add explicit `record_` overrides for booleans and (un)signed integers so we can have the most optimized conversions instead of the default debug print-based approach.

Reply:
Do you want to create an issue from this comment maybe?

Reply:
Filed #90 for this.  Good call.